iroha/docs/guides/develop-iroha-module.md

```diff
@@ -0,0 +1,148 @@
+# How-to develop a new Iroha Module
+
+When you need to add some functionality to the Iroha use this guide to develop a new Iroha Module.
```

Comment:
When you need to add some functionality to Iroha, use this guide to develop a new Iroha Module.